azurerm/internal/services/containers/data_source_kubernetes_cluster.go

```diff
@@ -231,6 +231,29 @@ func dataSourceArmKubernetesCluster() *schema.Resource {
 				Computed: true,
 			},
 
+			"identity": {
+				Type:     schema.TypeList,
+				Optional: true,
+				ForceNew: true,
```

Comment:
this is a Data Source - so this wants to be Computed only (and not Optional/ForceNew)